pyccl/tracers.py

```diff
@@ -518,8 +678,17 @@ def __init__(self, cosmo, dndz, has_shear=True, ia_bias=None,
                 a = tmp_a
             # Reverse order for increasing a
             t_a = (1./(1+z_a[::-1]), a[::-1])
-            self.add_tracer(cosmo, kernel=kernel_i, transfer_a=t_a,
-                            der_bessel=-1, der_angles=2)
+            if (cosmo['sigma_0'] == 0):
+                # GR case
+                self.add_tracer(cosmo, kernel=kernel_i, transfer_a=t_a,
+                                der_bessel=-1, der_angles=2)
+            else:
+                # MG case
+                k = np.logspace(-5, -0.7)
+                mg_transfer = self._get_MG_transfer_function(cosmo, z_a, k)
```

Comment:
Since intrinsic alignments trace the matter potential rather than the "light" potential, should we actually be modifying this tracer function?

Reply:
And the converse of this: shouldn't the magnification part of number counts be modified (since it's related to lensing)?

Reply:
Agreed. All fixed.